hack/dockerfile/install/containerd.installer

```diff
@@ -19,7 +19,7 @@ install_containerd() {
 		export EXTRA_LDFLAGS='-extldflags "-fno-PIC -static"'
 
 		# Reset build flags to nothing if we want a dynbinary
-		if [ "$1" == "dynamic" ]; then
+		if [[ "$1" == "dynamic" ]]; then
```

Comment:
Hm, I'd rather not change these if not needed (see the discussion on https://github.com/moby/moby/pull/38424 and https://github.com/moby/moby/pull/38527)

The bug here is actually that `==` is a bashism, and POSIX uses a _single_ `=` (see http://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128_17)

Changing to this should fix the issue:

```suggestion
		if [ "$1" = "dynamic" ]; then
```

Reply:
thanks, changed as comment, PTAL